mlflow/go/pkg/store/sql/store.go

```diff
@@ -496,6 +497,332 @@ func (s Store) DeleteExperiment(id string) *contract.Error {
 	return nil
 }
 
+const batchSize = 100
+
+type conflictedParam struct {
+	RunID    string
+	Key      string
+	OldValue string
+	NewValue string
+}
+
+//nolint:exhaustruct
+func (s Store) logParamsWithTransaction(
```

Comment:
Wowee this file is big! Is there any way to start teasing this apart so it all looks a little more SRPish?

Reply:
Yep, that is the plan I believe. We plan to shuffle some more things around after this endpoint.